src/py/flwr/superexec/simulation.py

```diff
@@ -129,7 +147,9 @@ def start_run(
             ]
 
             if override_config:
-                command.extend(["--run-config", f"{override_config}"])
+                override_config_str = self._user_config_to_str(override_config)
+                print(override_config_str)
```

Comment:
Let's remove the `print` and make `_user_config_to_str ` a function instead of a method